website/cue/reference/components/sources/syslog.cue

```diff
@@ -97,6 +97,22 @@ components: sources: syslog: {
 				examples: ["/path/to/socket"]
 			}
 		}
+		socket_file_mode: sources.socket.configuration.socket_file_mode
+            common: false
+			description: """
+				Unix file mode bits to be applied to the unix socket file
+				as its designated file permissions.
+				Note that the file mode value can be specified in any numeric format
+				supported by TOML, but it'd be more intuitive to use an octal number.
+				"""
+			relevant_when: "mode = `unix`"
+			required:      false
+			type: uint: {
+				default: null
+				unit:    null
+				examples: [0o777, 0o600, 508]
+			}
+		}
```

Comment:
Ah, you can delete these bits, it'll inherit. Apologies, I think I messed that up when creating the diff.